nilearn/maskers/nifti_maps_masker.py

```diff
@@ -74,6 +77,18 @@ class NiftiMapsMasker(BaseMasker, _utils.CacheMixin):
     %(memory)s
     %(memory_level)s
     %(verbose0)s
+
+    keep_masked_maps : :obj:`bool`, optional
```

Comment:
It's really unclear for now but we can combine some other information from `_trim_maps` to make this clearer. Here's my understanding of the behaviour and we can iterate on this:

When user supply a mask through parameter `mask_img`, the atlas will be masked by `mask_img` before any sign extraction happens. Some maps in the atlas will contain no brain coverage after applying the mask, resulting a map containing only zeroes (i.e. an invalid map for signal extraction). 

If `True`, masked atlas with these invalid maps will be retained in the output, resulting the correspondent time series containing zeros only. If `False`, the invalid maps will be removed from the trimmed altas, hence no empty time series will be present in the output.

Here I copy the docstrings from `_trim_maps` for reference:
```
Crop maps using a mask.

No consistency check is performed (esp. on affine). Every required check
must be performed before calling this function.

Parameters
----------
maps : :class:`numpy.ndarray`
    Set of maps, defining some regions.

mask : :class:`numpy.ndarray`
    Definition of a mask. The shape must match that of a single map.

keep_empty : :obj:`bool`, optional
    If False, maps that lie completely outside the mask are dropped from
    the output. If True, they are kept, meaning that maps that are
    completely zero can occur in the output.
    Default=False.

order : "F" or "C", optional
    Ordering of the output maps array (trimmed_maps).
    Default="F".

Returns
-------
trimmed_maps : :class:``numpy.ndarray`
    New set of maps, computed as intersection of each input map and mask.
    Empty maps are discarded if keep_empty is False, thus the number of
    output maps is not necessarily the same as the number of input maps.
    shape: mask.shape + (output maps number,). Data ordering depends
    on the "order" parameter.

maps_mask : :class:`numpy.ndarray`
    Union of all output maps supports. One non-zero value in this
    array guarantees that there is at least one output map that is
    non-zero at this voxel.
    shape: mask.shape. Order is always C.

indices : :class:`numpy.ndarray`
    Indices of regions that have an non-empty intersection with the
    given mask. len(indices) == trimmed_maps.shape[-1].

```